src/app/tool-editor/sections/arguments/argument-inspector.component.ts

```diff
@@ -0,0 +1,83 @@
+import {Subject} from "rxjs";
+import {Component, Input, Output} from "@angular/core";
+import {FormBuilder, FormGroup, FormControl} from "@angular/forms";
+import {CommandArgumentModel} from "cwlts/models/d2sb";
+import {ComponentBase} from "../../../components/common/component-base";
+
+@Component({
+    selector: "ct-argument-inspector",
+    template: `
+<ct-form-panel [borderless]="true" [collapsed]="false">
+
+        <span class="tc-header">
+            Basic
```

Comment:
You can remove the `ct-form-panel` completely. It's the only section of the inspector, so it isn't necessary.